js/apps/admin-ui/src/user/EditUser.tsx

```diff
@@ -115,6 +116,30 @@ export default function EditUser() {
         isUserProfileEnabled ? user.userProfileMetadata : undefined,
       );
 
+      if (isUserProfileEnabled) {
+        user.unmanagedAttributes = unmanagedAttributes;
```

Comment:
Why set this state on the user? Would it not be better to keep this in a separate `useState()` that to mutate the user?

Reply:
IIRC, this is because the user attribute component relies on the field on the user object set to the form. This is not different than how attributes work.

Reply:
Except attributes are simply converted to 'form state', meaning they are represented as state that can be correctly interpreted. They are however not fetched separately and merged with another representation.

If we need to keep track of the 'unmanagedAttributes' state, I would say it deserves it's own state that can be passed as a prop separately to any child components.